schema/tables/cryptoinfo.yml

```diff
@@ -0,0 +1,13 @@
+name: cryptoinfo
+description: TODO
+evented: false
+notes: This table is not a core osquery table. It is included as part of fleetd, the osquery manager from Fleet. Code based on work by [Kolide](https://github.com/kolide/launcher).
+platforms:
+  - darwin
+  - windows
+  - linux
+columns:
+  - name: name
+    # description: TODO
```

Comment:
Looks like this one still needs a description

Reply:
The [`option_roms_allowed` column of the `firmwarepasswd` table ](https://github.com/fleetdm/fleet/pull/15046/files#diff-119dc4e0bb0d1bb16cc09c851a96605d5aac37df0559e83a936aa6aa8cd885acR9)also has a commented-out description. Once these have descriptions then this PR should pass the website test.